nurse_scheduling.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This code includes an implementation of the algorithm described in Ikeda, 
+# K., # Nakamura, Y. & Humble, T.S. Application of Quantum Annealing to Nurse 
+# Scheduling Problem. Sci Rep 9, 12837 (2019). 
+# https://doi.org/10.1038/s41598-019-49172-3, © The Author(s) 2019, use of 
+# which is licensed under a Creative Commons Attribution 4.0 International 
+# License (To view a copy of this license, visit 
+# http://creativecommons.org/licenses/by/4.0/).
+
+from dwave.system import LeapHybridSampler
+from dimod import BinaryQuadraticModel
+
+# count nurses n = 1 ... N_NURSES
+# count scheduling days as d = 1 ... N_DAYS
```

Comment:
On this one, if it's OK with you, I would like to leave it as is. I find block comments harder to read.

Reply:
Typo? Because I think we are both saying that block comments aren't necessary in this case, however, you are also saying that you would prefer to leave the block comment as is.

Reply:
Oops. It is definitely a typo. I meant, I find "inline" comments hard to read. I prefer the block comments, for the eyes. It was a question of whether that's OK with you.

Reply:
For the case here, since your comment is short and is only defining that single variable, it should really be in-line. This way, users can read the block comments and quickly skim through the code. When people need to know details like the specific use of the variable, they can read the in-line comment

Reply:
OK, I changed them. I still find this hard to read, but perhaps I need to "get with the program" in the 21st century. :)